Add protocols as top-level directory with third_party protocols (#13)
robocin/ssl-core: protocols/protocols/third_party/README.md

# RoboCup's Small Size League Messages

This directory contains the libraries needed for a RoboCup Small Size League game.

## Protobufs

> The `.proto` files in this directory are named differently from the original files. The mapping between the files
> described here and the original files are shown below:

Reviewer: ```suggestion
The `.proto` files in this directory are named differently from the original files and the mapping between them and and the original ones is shown below:
```
